application/controllers/ManageUserDevicesController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/* Icinga Web 2 | (c) 2020 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Controllers;
+
+use Icinga\Web\RememberMe;
+use Icinga\Web\RememberMeUserList;
+use Icinga\Web\RememberMeUserDevicesList;
+use ipl\Web\Compat\CompatController;
+use ipl\Web\Url;
+
+/**
+ * ManageUserDevicesController
+ *
+ * you need 'application/sessions' permission to use this controller
+ */
+class ManageUserDevicesController extends CompatController
+{
+    public function indexAction()
+    {
+        $this->getTabs()
+            ->add(
+                'manage-user-devices',
+                array(
+                    'title'             => $this->translate('List of users who stay logged in'),
+                    'label'             => $this->translate('Users'),
+                    'url'               => 'manage-user-devices',
+                    'data-base-target'  => '_self'
+                )
+            )->activate('manage-user-devices');
+
+        $this->assertPermission('application/sessions');
```

Comment:
Check the permission in method `init`. This way it's done for every action. Currently only the index action requires the permission.